test/Microsoft.Sbom.Tool.Tests/IntegrationTests.cs

```diff
@@ -229,9 +257,9 @@ private string CreateTestFolder()
         return testFolderPath;
     }
 
-    private static string AppendFullManifestFolderPath(string manifestDir)
+    private static string AppendFullManifestFolderPath(string manifestDir, string spdxVersion = "2.2")
```

Comment:
Since the caller's default for `spdxVersion` is null and you're checking it below, you can just use null here and only have 1 line that determines the default version.
```suggestion
    private static string AppendFullManifestFolderPath(string manifestDir, string spdxVersion = null)
```